scraper/circleci_scraper.go

```diff
@@ -74,81 +80,126 @@ func circleciApiCall(url string) []byte {
 	return body
 }
 
-func getPipelines(branch string) Pipelines {
-	var pipelines Pipelines
+func getPipelines(branch string) []Pipeline {
+	body := circleciApiCall(fmt.Sprintf("project/gh/filecoin-project/lotus/pipeline?branch=%s", branch))
```

Comment:
We want project and repo here besides the branch, since we will use this on different repo's i suppose?

Reply:
Yep, I'll change it 👍